fabric.ts

```diff
@@ -13,12 +13,15 @@ export { Canvas } from './src/canvas/Canvas';
 export { Point } from './src/Point';
 export { Intersection } from './src/Intersection';
 export { Color } from './src/color/Color';
+export * from './src/color/typedefs';
 
 export { Gradient } from './src/gradient/Gradient';
 export * from './src/Pattern';
 export { Shadow } from './src/Shadow';
 
 export { BaseBrush } from './src/brushes/BaseBrush';
+export * from './src/brushes/typedefs';
+
 export { PencilBrush } from './src/brushes/PencilBrush';
 export { CircleBrush } from './src/brushes/CircleBrush';
 export { SprayBrush } from './src/brushes/SprayBrush';
```

Comment:
what is the decision regarding index files?

Reply:
If it doesn't make my life simpler i don't add one.
in this case i could write a line to export types and i did.

Reply:
i think i can see a typedef file as the index file of the types alone.
Since i try to differentiate types from code exports, in this  case a file alone serve the purpose well